api/TicketMasterAPI.js

```diff
@@ -0,0 +1,412 @@
+
+const fs = require("fs")
+const bodyParser = require("body-parser")
+const cors = require('cors');
+const express = require('express')
+const app = express()
+const port = 8080
+
+app.use(bodyParser.json())
+app.use(cors());
+
+const knex = require('knex')(require('./knexfile.js')['development']);
+/*
+var serverCookies = [];
+function validateCookie(req, res, next) {
+	if (!req.cookies) {
+		let {cookies} = req;
+		if ('email' in cookies && 'password' in cookies) {
+			
+			if (serverCookies.includes(cookies.email) && serverCookies.includes(cookies.password)) {
+				next();
+			} else {
+				res.status(403).send({'succeeded': 'false', 'error': 'cookie mismatch on server'});
+			}
+		} else {
+			res.status(403).send({'succeeded': 'false', 'error': 'no cookie'});
+		}
+	}
+	next();
+}
+*/
+app.post('/auth', (req, res) => {
```

Comment:
There's a lot going on in this request, but it mostly seems to be fetching the groups in the DB that are accessible to a user.  This would generally be a GET request, rather than a POST, which tends to denote creating a NEW record in the DB, i.e. making a new user or making a new ticket.